Libraries/etl/memory.h

```diff
@@ -45,6 +45,8 @@ SOFTWARE.
 
 #if defined(ETL_IN_UNIT_TEST) || ETL_USING_STL
   #include <memory>
+#include <cstring>
```

Comment:
this was needed to fix a compile error? seems weird we'd have to add includes to etl headers